docs/development.rst

```diff
@@ -133,6 +133,137 @@ To get an idea about the size distribution of our react codebase you can run::
 
 This will open an interactively explorable report in your browser.
 
+Add a React page
+----------------
+
+First we need to set up a few things in Django:
+
+#. Create a new Jinja template that our view will render (we'll use ``example.html``). Paste the following code in ``example.html`` to get started::
+
+    {% extends "react_base.html" %}
+
+    {% block title %}{{ page_title(_('My Page Title')) }}{% endblock %}
+
+    {% block content %}This works!{% endblock %}
+
+    {# This overrides the Jinja footer, since we will be using the React footer instead. Without this line, two footers will render. #}
+    {% block footer %}{% endblock %}
+
+#. In ``views.py``, render the Jinja template we just created::
```

Comment:
Continuing on from the above comment, let's be specific as to which `views.py` we are editing.

Maybe we can use the `users` app under `kuma/` in this example?

Reply:
I'm going to use `/dashboards/`, because the folder structure is a bit simpler than `/users/` (specifically the jinja2 folder)